src/services/sync/users.ts

```diff
@@ -0,0 +1,23 @@
+import { ToastAndroid } from "react-native";
+import { UserClient } from "../api/users";
+import { LocalDatabase } from "../db/db"
+
+const uploadUsersData = async () => {
+
+    const dbClient = new LocalDatabase();
+    const apiClient = new UserClient();
+    
+    if (!dbClient.users) {
+        ToastAndroid.show("Something went wrong. please try again letter", 10);
+        return;
+    }
+    try {
+        const users = await dbClient.users.getLocalUsers(0, -1, false);
```

Comment:
we should only try to make api calls for the new ones, not all the ones. Plus, we should make use of last timestamp when things were synced. and use that.